packages/cloud_asset_inventory/manifest.yml

```
@@ -11,7 +11,7 @@ categories:
   - cloudsecurity_cdr
 conditions:
   kibana:
-    version: "^8.16.0"
+    version: ">=8.16.0"
```

@kubasobon what was the motivation of this change? This means that this package supports any future major, what cannot be guaranteed. This is why we prefer to use `^`.

If you would like to indicate that this package is supported in 9.x, it is safer to use:
```suggestion
    version: "^8.16.0 || ^9.0.0"
```